packages/plugin-npm/sources/npmHttpUtils.ts

```diff
@@ -0,0 +1,28 @@
+import {Configuration, httpUtils, Ident} from '@berry/core';
+
+export function get(url: string, ident: Ident, configuration: Configuration) {
+  const headers: {[headerName: string]: string} = {};
+
+  const auth = getAuthenticationHeader(ident, configuration);
+  if (auth)
+    headers.authorization = auth;
+
+  return httpUtils.get(
+      url,
+      configuration,
+      {headers},
+  );
+}
+
+function getAuthenticationHeader(ident: Ident, configuration: Configuration) {
+  if (!configuration.get(`npmAlwaysAuth`) && !ident.scope)
```

Comment:
At the moment this implies you can no longer fetch scoped packages without authentication. That's obviously wrong&hellip;

Reply:
Btw, return `null` rather than `undefined` (undefined should imo only be generated by the js engine)